nexus/db-model/src/tuf_repo.rs

```diff
@@ -87,6 +87,7 @@ impl TufRepoDescription {
 pub struct TufRepo {
     pub id: DbTypedUuid<TufRepoKind>,
     pub time_created: DateTime<Utc>,
+    pub time_pruned: Option<DateTime<Utc>>,
```

Comment:
There's some Diesel-related context where the order of things matters.  I can't remember if it's the order of fields in this struct or something else.

If you can correctly read a value out of the table in Rust, then it should be fine.

Reply:
I think it's the order of fields in `schema.rs`?